### ISSUE TITLE
Additions to MCB

### ISSUE THREAD

**Issue by cgc** [cgc/rnd#4]
- Adding UI to permit creating a new agent, training an agent more, hand-tuning an agent, or setting hyper-parameters of an agent.
- Making things a little more deterministic, particularly in a way that's tweakable via the UI.
- Consider other ways of training agent besides policy gradients + RL. [overview](http://learning.mpi-sws.org/mlss2016/slides/2016-MLSS-RL.pdf)
    - Cross-entropy method. papers on CEM: [0](http://nipg.inf.elte.hu/publications/szita06learning.pdf), [1](https://esc.fnwi.uva.nl/thesis/centraal/files/f2110275396.pdf), [keras-rl CEM implementation](https://github.com/matthiasplappert/keras-rl/blob/master/rl/agents/cem.py)
    - Policy gradient with continuous action space. post [0](http://pemami4911.github.io/blog/2016/08/21/ddpg-rl.html)
    - TRPO? Q-learning? Actor-Critic?

**Comment by cgc** [comment]
Should consider using `deeplearn.js` instead of synaptic, which has GPU support. https://deeplearnjs.org
